python/podio_gen/generator_utils.py

```diff
@@ -29,6 +29,37 @@ def _prefix_name(name, prefix):
   return name
 
 
+def get_julia_type(cpp_type=None, is_array=False, array_type=None, array_size=None):
```

Comment:
```suggestion
def get_julia_type(cpp_type, is_array=False, array_type=None, array_size=None):
```
Since this doesn't do anything if `cpp_type` is actually `None`. Better make not passing one an error rather than going on and producing something that will most likely crash down the line.